src/mgr/QueryParams.cc

```diff
@@ -65,61 +69,69 @@ Mgr::QueryParams::find(const String& name) const
     return iter;
 }
 
-bool
-Mgr::QueryParams::ParseParam(const String& paramStr, Param& param)
+/// Parses the value part of a "param=value" URL section.
+/// Value can be a comma-separated list of integers or an opaque string.
+/// \note opaque string may be a list with a non-integer (e.g., "1,2,3,z")
+Mgr::QueryParam::Pointer
+ParseParamValue(SBuf &rawValue)
```

Comment:
Please keep the parameter constant if possible.

```suggestion
ParseParamValue(const SBuf &rawValue)
```